app/models/reports/region_summary_schema.rb

```diff
@@ -350,9 +354,9 @@ def visited_without_bp_taken(with_ltfu: false)
 
     memoize def diabetes_patients_with_bs_taken
       region_period_cached_query(__method__) do |entry|
-        diabetes_under_care(:bs_below_200)[entry.region.slug][entry.period] +
-          diabetes_under_care(:bs_200_to_300)[entry.region.slug][entry.period] +
-          diabetes_under_care(:bs_over_300)[entry.region.slug][entry.period]
+        adjusted_diabetes_under_care(:bs_below_200)[entry.region.slug][entry.period] +
+          adjusted_diabetes_under_care(:bs_200_to_300)[entry.region.slug][entry.period] +
+          adjusted_diabetes_under_care(:bs_over_300)[entry.region.slug][entry.period]
```

Comment:
Consumers who use `region_summary_schema` need `adjusted_` data from the table but do not want to mention it every time they use it. Hence we are not prefixing the methods with `adjusted_`. Only the ` adjusted_diabetes_under_care` method is out of this convention.